src/controllers/expenses.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+const { getUsers } = require('./users');
+const { getNextId } = require('../getNextId');
+
```

Comment:
Create a service for logic & move logic witch related to the `service`
Split your logic here & in the `users.js` file